draft-js-plugins-editor/CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - `decorators` don't decorate plugins anymore.
 
+### Changed
+
+- *Breaking change* The `plugins` and `decorators` editor props need to be an Immutable.List()
```

Comment:
I like it because it resembles the Draft.js API and the other hand it makes things more complicated.

Right now I have the feeling either we keep the current behaviour + add `decoratorFromPlugins` for advanced usage or we set the decorators on every new editorState and test that it's not causing performance issues especially with a lot of content.